packages/@aws-cdk/core/lib/secret-value.ts

```diff
@@ -1,35 +1,72 @@
+import { CHECK_SECRET_USAGE } from '@aws-cdk/cx-api';
 import { CfnDynamicReference, CfnDynamicReferenceService } from './cfn-dynamic-reference';
 import { CfnParameter } from './cfn-parameter';
+import { CfnResource } from './cfn-resource';
+import { FeatureFlags } from './feature-flags';
+import { CfnReference } from './private/cfn-reference';
 import { Intrinsic } from './private/intrinsic';
-import { Token } from './token';
+import { IResolveContext } from './resolvable';
+import { Token, Tokenization } from './token';
 
 /**
  * Work with secret values in the CDK
  *
- * Secret values in the CDK (such as those retrieved from SecretsManager) are
- * represented as regular strings, just like other values that are only
- * available at deployment time.
+ * Constructs that need secrets will declare parameters of type `SecretValue`.
  *
- * To help you avoid accidental mistakes which would lead to you putting your
- * secret values directly into a CloudFormation template, constructs that take
- * secret values will not allow you to pass in a literal secret value. They do
- * so by calling `Secret.assertSafeSecret()`.
+ * The actual values of these secrets should not be committed to your
+ * repository, or even end up in the synthesized template. Instead, they should
+ * be stored in an external system like AWS Secrets Manager or SSM Parameter
+ * Store, and you reference them by calling `SecretValue.secretsManager()` or
+ * `SecretValue.ssmSecure()`.
```

Comment:
Remove passive voice, and make sentence structure consistent.
```suggestion
 * repository, or even end up in the synthesized CloudFormation template. Instead, you should
 * store them in an external system like AWS Secrets Manager or SSM Parameter
 * Store, and you can reference them by calling `SecretValue.secretsManager()` or
 * `SecretValue.ssmSecure()`.
```